QuickDeployment/QuickDeploymentDesktopReadme.md

```diff
@@ -4,56 +4,62 @@ This system has been pre-configured as a fully functioning C4B environment.
 
 <!-- TOC -->
 
-- [Summary](#summary)
-- [Create a license package](#create-a-license-package)
-- [Enable Central Management](#enable-central-management)
-- [Server Information](#server-information)
-  - [Nexus Repository](#nexus-repository)
-    - [Changing the API Key](#changing-the-api-key)
-    - [Choco apikey Command](#choco-apikey-command)
-  - [Jenkins](#jenkins)
-  - [Chocolatey Central Management](#chocolatey-central-management)
-  - [Firewall ports](#firewall-ports)
-  - [Browser considerations](#browser-considerations)
-  - [SSL Information](#ssl-information)
-- [Client installations](#client-installations)
-- [Licensing this VM](#licensing-this-vm)
-- [Package Internalization](#package-internalization)
+- [Thanks for trying Chocolatey For Business!](#thanks-for-trying-chocolatey-for-business)
```

Comment:
got to fix this bit. Maybe in the TOC comments on line 5 we can `depthFrom:2` (IIRC)

Reply:
My Markdown extension doesn't respect that, interestingly enough. No biggie, I'll add it in for other folks.

Reply:
We may need to look it up

Reply:
https://github.com/AlanWalk/markdown-toc#32-insert-header-number-sections

Reply:
`<!-- TOC depthFrom:2 -->`

Reply:
Ah, it's specific to that extension. Gotcha, ty!